bitsail-connectors/bitsail-connectors-legacy/bitsail-connector-hive/src/main/java/com/bytedance/bitsail/connector/legacy/hive/sink/HiveOutputFormat.java

```diff
@@ -160,7 +161,12 @@ protected void validateParameters() {
     hiveWritableExtractorType =
         HiveWritableExtractorType.valueOf(outputSliceConfig.get(HiveConversionOptions.WRITABLE_EXTRACTOR_TYPE).toUpperCase());
     partition = outputSliceConfig.getNecessaryOption(HiveWriterOptions.PARTITION, HiveParqueFormatErrorCode.REQUIRED_VALUE);
-    columns = outputSliceConfig.get(WriterOptions.BaseWriterOptions.COLUMNS);
+
+    if (outputSliceConfig.fieldExists(HiveWriterOptions.COLUMNS)) {
+      columns = outputSliceConfig.getNecessaryOption(
```

Comment:
Could you move these lines to 405 line around, so that the method for getting columns are together.

Reply:
Thanks for your suggestion, fixed.